api/src/main/resources/client_api.yaml

```diff
@@ -74,6 +74,70 @@ paths:
           schema:
             $ref: "#/definitions/MaterializeCohortResponse"
 
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/concepts:
+      post:
+        tags:
+          - concepts
+
+        description: >
+          Searches for concepts in concept table by name, and optionally filter on
+          domain, vocabulary ID, or standard concept status. Uses the CDR version affiliated
+          with the workspace specified in the path.
+
+        operationId: "searchConcepts"
+        parameters:
+          - in: path
+            name: workspaceNamespace
+            description: The namespace of the workspace for which concepts are being looked up.
+            required: true
+            type: string
+          - in: path
+            name: workspaceId
+            description: The ID of the workspace for which concepts are being looked up.
+            required: true
+            type: string
+          - in: body
```

Comment:
If you're going with POST, please make this into a JSON body with a request model definition as we have elsewhere. For consistency, but also this makes the swagger codegen better (Java controller method currently has N parameters).

Reply:
Done.